vendor/knative.dev/eventing/test/upgrade/prober/sender.go

```diff
@@ -94,6 +96,12 @@ func (p *prober) deploySender() {
 func (p *prober) removeSender() {
 	p.log.Info("Remove of sender deployment: ", sender.Name)
 
+	// Before removing the sender same logs.
```

Comment:
@pierDipi This is actually a good feature! Could you send this upstream ?

Reply:
yes, done https://github.com/knative/eventing/pull/6384/files